CHANGELOG.md

```diff
@@ -5,9 +5,9 @@
 * Remove balance from transaction header
 * Add checkbox icon button to activate selection mode in transactions page and search page
 * Improve ui for no result in search page
+* Recategorizing several transactions is faster
```

Comment:
Are we talking about "faster performance" or "fewer user actions needed"?

nit: this is not related to the Android PR
nit: on my mind "several" means "multiple but not many", should we use "multiple" or "many" instead?

Reply:
faster performance. Changed in `Recategorizing multiple transactions is much faster with better performance`